homeassistant/components/media_player/plex.py

```diff
@@ -189,6 +192,9 @@ def update_devices():
         if new_plex_clients:
             add_devices_callback(new_plex_clients)
 
+        for cid in hass.data[PLEX_DATA].keys():
```

Comment:
Why didn't you use this loop: https://github.com/ryanm101/home-assistant/blob/4d8a8606499b42936d7ad0ca9d77cf13b60a725e/homeassistant/components/media_player/plex.py#L187-L190

Reply:
yes this could be moved.